core/equations.gms

```diff
@@ -939,11 +923,6 @@ q_ccsShare(t,regi) ..
 ***---------------------------------------------------------------------------
 *' Definition of the CCS transformation chain:
 ***---------------------------------------------------------------------------
-*** no effect while CCS chain is limited to just one step (ccsinje)
-q_transCCS(t,regi,ccs2te(enty,enty2,te),ccs2te2(enty2,enty3,te2),rlf)$teCCS2rlf(te2,rlf)..
```

Comment:
@tabeado @amerfort Can this equation be removed, as the comment indicates, or does it matter in any currently used scenario?